gwsumm/data/coherence.py

```diff
@@ -131,8 +131,38 @@ def _get_coherence_spectrogram(channel_pair, segments, config=None,
         sampling = None
 
     # initialize component lists if they don't exist yet
+    # store compnents in a backup variable
+    coherence_bkp = {}
     for ck in ckeys:
         globalv.COHERENCE_COMPONENTS.setdefault(ck, SpectrogramList())
+        coherence_bkp[ck] = globalv.COHERENCE_COMPONENTS[ck]
```

Comment:
Not sure if this makes the code more readable
```suggestion
        coherence_bkp[ck] = globalv.COHERENCE_COMPONENTS.get(ck)
```

Reply:
As you a suggesting the use of `.get` method, I would suggest also add `SpectrogramList()` as the default value, instead of leaving `None` as default.

Reply:
Sure, sounds good

Reply:
Applied this suggestion in new commit.